services/order.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+from django.db import transaction
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None
+                 ) -> QuerySet:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+
+    if date:
+        order.created_at = date
+        order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(order=order,
+                              movie_session_id=ticket["movie_session"],
+                              row=ticket["row"],
+                              seat=ticket["seat"]
+                              )
+    return order
+
+def get_orders(username: str = None) -> QuerySet:

```

Comment:
The function `get_orders` could be improved by adding type hints for its return type. This is a code quality issue.